main/externalData/storeApi.ts

```diff
@@ -1,11 +1,23 @@
 import store from '../store'
 import { NATIVE_CURRENCY } from '../../resources/constants'
 
-import type { Chain, Token, Rate, InventoryAsset, Inventory, TokenBalance } from '../store/state/types'
+import type {
+  Chain,
+  Token,
+  Rate,
+  InventoryAsset,
+  Inventory,
+  TokenBalance,
+  Account
+} from '../store/state/types'
 
 export const storeApi = {
   // Accounts
-  getActiveAddress: () => (store('selected.current') || '') as Address,
+  getActiveAddress: () => {
+    const accounts = Object.values(store('main.accounts')) as Account[]
```

Comment:
once you know the type of `main.accounts` I don't think we need anymore casting:

```suggestion
    const accounts = Object.values(store('main.accounts')) as Account[]
    return accounts.find((acct: Account) => acct.active)?.id || ''
```